packages/noco-docs/content/en/developer-resources/rest-apis.md

```diff
@@ -167,6 +167,7 @@ Currently, the default value for {orgs} is <b>noco</b>. Users will be able to ch
 | Meta | Post | utils | testConnection | /api/v1/db/meta/projects/connection/test |
 | Meta | Get | utils | appInfo | /api/v1/db/meta/nocodb/info |
 | Meta | Get | utils | appVersion | /api/v1/db/meta/nocodb/version |
+| Meta | Get | utils | appHealth | /api/v1/db/meta/nocodb/health |
```

Comment:
Path for health is universal. Doesn't have to be under db/meta/nocodb

Reply:
What about appInfo & appVersion?

Reply:
so it would be 

`/api/v1/db/meta/nocodb/version` -> `/api/v1/version`
`/api/v1/db/meta/nocodb/health` -> `/api/v1/health`

please endorse.